ros2_batch_job/osx_batch/__init__.py

```diff
@@ -36,16 +36,19 @@ def pre(self):
         else:
             warn('ccache does not appear to be installed; not modifying PATH')
         if 'LANG' not in os.environ:
+            warn('LANG unset; using default value')
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
         if 'OPENSSL_ROOT_DIR' not in os.environ:
+            warn('OPENSSL_ROOT_DIR not set; installing openssl')
```

Comment:
I think this is just **finding** openssl installed from brew but not performing the installation.

Reply:
Fixed text
1986152